server.js

```diff
@@ -129,6 +137,11 @@ function renderPage(appHtml, reactHelmet, canonicalUrl) {
         <title>"${meta.title}"</title>
         <meta name="description" content="${meta.description}">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        ${
+          env.STAGING_SERVER && (
```

Comment:
Unfortunately this `${ boolean && string}` syntax doesn't work here. It will also output true/false as string as part of the DOM 😞

Reply:
It shouldn't, because `true && something` is `something` in JS O_o

I'll have a look at what's going on because I use that pattern all over the place in side projects and always see it do the right thing =S

Reply:
I thought so too, but it outputs (see screenshot) on the screen if I set the env var to `True`

![image](https://user-images.githubusercontent.com/2896608/111343569-c9fca380-8638-11eb-8319-48cd4efbc4f2.png)

I haven't checked its source code but I wonder if it's `react-dom/server` doing some weird conversion on their end.

Reply:
Ah found the issue! 😂 We just need to wrap `<meta name="googlebot" content="noindex, nofollow, noarchive" />` in quotes since we are dealing things in strings and not React nodes here.

Reply:
haha, omg well spotted =D